standalone/src/init.ts

```diff
@@ -282,6 +282,9 @@ export default async function init(cfg: InitConfig) {
         case `custom_${pluginId}_host_lower_hand`:
             handleHostLowerHand(store, ev as WebSocketMessage);
             break;
+        case `custom_${pluginId}_host_removed`:
+            handleHostRemoved(store, ev as WebSocketMessage);
```

Comment:
Eventually, I'd expect us to update our [Go client implementation](https://github.com/mattermost/rtcd/tree/master/client) to handle these events accordingly.

Reply:
sure: https://mattermost.atlassian.net/browse/MM-58171